release-notes/9.0/README.md

```diff
@@ -9,10 +9,33 @@
 - [Known Issues](known-issues.md)
 - [Installation instructions](install.md)
 
-## Releases
+## Releases & Release Notes

```

Comment:
```suggestion
## Release Notes

```

I think "release notes" on its own is sufficient to get the idea across.

Reply:
LGTM, i was going back and forth on it